tests/integration-tests/conftest.py

```diff
@@ -163,15 +163,32 @@ def pytest_configure(config):
     _setup_custom_logger(config.getoption("tests_log_file"))
 
 
+def pytest_sessionstart(session):
+    # The number of seconds before a connection to the instance metadata service should time out.
+    # When attempting to retrieve credentials on an Amazon EC2 instance that is configured with an IAM role,
+    # a connection to the instance metadata service will time out after 1 second by default. If you know you're
+    # running on an EC2 instance with an IAM role configured, you can increase this value if needed.
+    os.environ["AWS_METADATA_SERVICE_TIMEOUT"] = "5"
```

Comment:
out of curiosity -- what is the default here?

Reply:
1 second. I left a note in the comment above:
>a connection to the instance metadata service will time out after 1 second by default.

Reply:
Ah -- missed that while skimming. :+1: